packages/playwright/src/index.ts

```
@@ -0,0 +1,83 @@
+ * });
+ * ```
+ */
+export async function injectChaos(page: Page, config: ChaosConfig): Promise<void> {
+  const scriptContent = getCoreScript();
+
+  await page.addInitScript((args: { config: unknown; scriptContent: string }) => {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    const win = globalThis as any;
+    win.__CHAOS_CONFIG__ = args.config;
+    // eval executes synchronously, ensuring fetch/XHR are patched before any app code
+    (0, eval)(args.scriptContent);
```

Biome (2.4.9)

[error] eval() exposes to security risks and performance issues.

(lint/security/noGlobalEval)